src/frontend/utils/tekstformatering.ts

```diff
@@ -38,3 +38,15 @@ export const tekstEllerKode = <T extends string>(
     mapping: Record<T, string>,
     kode?: T
 ): string | undefined => (kode && mapping[kode]) || kode;
+
+export enum SvarJaNei {
```

Comment:
Denne typen er koblet til VilkårPeriode, så føler ikke at det er riktig å flytte den til `utils/teksformattering`
Tanken med `teksformattering` er "global tekstformattering"

Reply:
Men nå som vi skal begynne å bruke den så er det vel riktig at den flyttes til en "felles-utils"-fil? Eller skal FaktaUtdanning ha sin egen "JaNei"-implementasjon?

Reply:
Typen for fakta som er brukt er `JaNei`, der finnes det allerede en mapping som er `jaNeiTilTekst`

Reply:
Fixed: cecea6c9784bc33e9c2af180bd53a703fd53a256